### ISSUE TITLE
Json Editor 

### ISSUE THREAD

**Issue by dmushkov** [stoplightio/json-schema-tree#3]
Is there any chance to have Json Editor example ?

**Comment by P0lip** [comment]
Ooops, I didn't see this one. My apologies.
Do you mean something similar to our [`@stoplight/json-schema-viewer`](https://github.com/stoplightio/json-schema-viewer)?

**Comment by dmushkov** [comment]
@P0lip yes it will be good to see some example not for viewer but for editor ....

**Comment by P0lip** [comment]
Unfortunately, we don't have anything I could share at the moment :disappointed: 
Our json-schema-editor is a private repository, and I cannot share the code.
